File: 20.heron_formulu.py
a=int(input("Birinci kenarı giriniz: "))
b=int(input("İkinci kenarı giriniz: "))
c=int(input("Üçüncü kenarı giriniz: "))
u=(a+b+c)/2
alan=(u*(u-a)*(u-b)*(u-c))**1/2
print("Alan: {:.2f}\n".format(alan))

File: c/10-20 arası örnekler/20_heron_formulu.c
#include <stdio.h>
#include <math.h>

int main()
{
    int a, b, c, u, alan, alan2;
    printf("Birinci kenari giriniz: ");
    scanf("%d", &a);
    printf("Ikinci kenari giriniz: ");
    scanf("%d", &b);
    printf("Ucuncu kenari giriniz: ");
    scanf("%d", &c);
    u = (a + b + c) / 2;
    alan = (u * (u - a) * (u - b) * (u - c));
    alan2 = pow(alan,0.5);
    printf("Sonuc: %d\n", alan2);
    return 0;
}